requirements.in

```diff
@@ -1,8 +1,8 @@
 -r requirements/base.txt
 -r requirements/bigquery_extras.txt
 -r requirements/dataflow_extras.txt
-pip>=22
```

Comment:
Isn't there any 22.x version that will work? It worked before 🤔

Reply:
For version 22.1 CI pipeline failed with same errors